pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+target-version = ['py36', 'py37']
+skip-string-normalization = true
```

Comment:
Can we add the max line length config to 79?

Reply:
haha sure thing :D